Web/starter_project_barbies/pages/team/Team.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import TeamsImage from '../../public/teams_page_img.svg'
+import Image from 'next/image'
+import TeamMemberCard from "@/components/team/TeamMemberCard";
+import { TeamMemberCardProps } from "@/types/team/TeamMemberCardProps";
+
+const DummyDataGenerator = (): TeamMemberCardProps[] => {
+  const data:TeamMemberCardProps[] = []
+  for (let i = 0; i < 10; i++) {
```

Comment:
why have this be a function?
why not define a dummy data json file and import that as a team member list? (you can put it inside a data folder in the root)